src/shadowbox/model/shadowsocks_server.ts

```diff
@@ -21,6 +21,10 @@ export interface ShadowsocksAccessKey {
 }
 
 export interface ShadowsocksServer {
+  // Annotates the Prometheus data metrics with countries.
```

Comment:
Let's have a enableAsnMetrics method instead. We can remove the country metrics, it's dead code.

Reply:
Done.